src/journey-maps/angular/services/map/two-d-three-d-service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Map as MaplibreMap } from 'maplibre-gl';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TwoDThreeDService {
+  /**
+   * If the level-switch feature is enabled by the client, then show only 3d layers (-lvl)
+   * If the level-switch feature is disabled by the client, then show only 2d layers (-2d)
+   */
+  public show2Dor3D(map: MaplibreMap, isLevelSwitchEnabled: boolean) {
+    this._setVisibility(map, '-2d', isLevelSwitchEnabled ? 'none' : 'visible');
+    this._setVisibility(map, '-lvl', isLevelSwitchEnabled ? 'visible' : 'none');
+  }
+
+  private _setVisibility(map: MaplibreMap, searchString1: string, newVisibility1: string) {
```

Comment:
```suggestion
  private _setVisibility(map: MaplibreMap, layerIdSuffix: string, visibility: 'visible' | 'none') {
```

Reply:
Done ✅

Reply:
Don't you like `visibility: 'visible' | 'none'` 🥲

Reply:
Sorry, das hatte ich nicht gesehen. Hab's gepushed.